CHANGELOG.md

```diff
@@ -26,6 +26,8 @@ an architecture not in the filter.
 ### Deprecated
 - `get_syscall_from_name`, use `ScmpSyscall::from_name*()`
 - `get_syscall_name_from_arch`, use `ScmpSyscall::get_name*()`
+- `ScmpFilterContext::get_no_new_privs_bit`, use `ScmpFilterContext::get_ctl_nnp`.
+- `ScmpFilterContext::set_no_new_privs_bit`, use `ScmpFilterContext::set_ctl_nnp`.
```

Comment:
```markdown
### Added
- `ScmpFilterContext::get_ctl_nnp` (replaces `ScmpFilterContext::get_no_new_privs_bit`).
- `ScmpFilterContext::set_ctl_nnp` (replaces `ScmpFilterContext::set_no_new_privs_bit`).
```

Reply:
I fixed it.

Reply:
I would rather have noted it in both places (added and deprecated). Anyway you could also use "changed" for renamed things. IDK if ther is the one right way to do it.